test/test_pure_wheel.py

```diff
@@ -35,8 +35,7 @@ def test(tmp_path, capfd):
     pure_python_project.generate(project_dir)
 
     with pytest.raises(subprocess.CalledProcessError):
-        actual_wheels = utils.cibuildwheel_run(project_dir)
-        print("produced wheels:", actual_wheels)
+        print(utils.cibuildwheel_run(project_dir))
```

Comment:
```suggestion
        print("produced wheels:", utils.cibuildwheel_run(project_dir))
```

This is only a debug printout if it fails, but missed this.